src/main/java/com/agency04/sbss/pizza/PizzaApp.java

```diff
@@ -1,9 +1,21 @@
 package com.agency04.sbss.pizza;
 
+import com.agency04.sbss.pizza.model.Margherita;
+import com.agency04.sbss.pizza.service.PizzaDeliveryService;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
 public class PizzaApp {
 
     public static void main(String[] args) {
         // Add some pizza code
+        ClassPathXmlApplicationContext context = new ClassPathXmlApplicationContext("spring-context.xml");
+
+        Margherita thePizza = new Margherita();
```

Comment:
Umjesto kreiranja objekta dohvatiti bean iz contexta. To odraditi za 2 vrste te također ispisati i sastojke.